tutorials/Tab_Navigation/README.md

```diff
@@ -309,3 +309,54 @@ fun main() = application {
 ```
 
 <img alt="reverse-order" src="focus-switcher.gif" height="480" />
+
+## Known problems
+
+### Tab key navigation work's bad with multiline TextField
+```Kotlin
+Column(
```

Comment:
This will be slighter better (to correspond the code I posted below):
```
Column {
    repeat(5) {
        var text by remember { mutableStateOf("Hello, World!") }

        OutlinedTextField(
            value = text,
            singleLine = false, // Pay attention here! Also, by default, singleLine is false.
            onValueChange = { text = it },
            modifier = Modifier.padding(8.dp)
        )
    }
}
```